src/group/service.ts

```diff
@@ -0,0 +1,69 @@
+import database from '../database';
+import { User } from '../user/types';
+
+type GroupRecord = {
+  id: number,
+  identity: string,
+  nickname: string,
+  created: number,
+  active: number,
+  type: number
+};
+
+const fieldNames = [
+  'id',
+  'identity',
+  'nickname',
+  'created',
+  'active',
+  'type'
+];
+
+/**
+ *  Checks if the user is a type group and returns true or false
+ */
+export async function isGroup(user: User): Promise<boolean> {
```

Comment:
This function still has the same issue. Why does it need to do a MySQL query to get it's information